### PR TITLE
Cold Start: add loading placeholders

### PR DESCRIPTION
**Pull request by bluefuton**
While waiting for the initial set of Cold Start recommendations to load, there's currently no indication that loading is happening.

This PR adds placeholder cards to let the user know that real cards are on their way.

screen shot 2016-06-20 at 20 53 02

Test live: https://calypso.live/?branch=add/reader/cold-start-loading-placeholders

### CODE REVIEW

File: client/reader/start/main.jsx [hunk 1]
```diff
@@ -55,8 +57,16 @@ const Start = React.createClass( {
 		page.redirect( '/' );
 	},
 
+	renderLoadingPlaceholders() {
+		const count = 4;
```
Comment (TooTallNate) [inline client/reader/start/main.jsx]: `config`, perhaps?

Comment (bluefuton) [inline client/reader/start/main.jsx]: You mean as a var name? I should probably go full verbose and do `numberOfPlaceholders` :)

Comment (TooTallNate) [inline client/reader/start/main.jsx]: I meant [config](https://github.com/Automattic/wp-calypso/blob/master/config). Back in the cloudup days we'd use config for things like this that are essentially magic numbers. `const numberOfPlaceholders = config('reader_number_of_placeholders')`.

Though taking a look at the current config is doesn't seem like we're using that for things like this, so probably nvm 🍤

Comment (bluefuton) [inline client/reader/start/main.jsx]: We probably should be though. I have always done that in other projects. Worth raising at a framework level?

**Comment by jancavan** [comment]
I just omitted the featured post heading, but other than that, everything LGTM 👍

**Comment by bluefuton** [comment]
Squashed! Now to 🚢 .

File: client/reader/start/main.jsx (hunk 1 above)
Comment (blowery) [inline client/reader/start/main.jsx]: I'll be the voice of dissent, at least for this particular value. 😈  

This isn't something that would change between deployment environments and moving it to config introduces a new layer of indirection that can break. Having it near where it's used is clear, easily understood, and unlikely to break in unexpected ways.

File: client/reader/start/main.jsx [hunk 2]
```diff
@@ -66,17 +76,19 @@ const Start = React.createClass( {
 					<p className="reader-start__description">{ this.translate( "We've suggested some sites below that you might enjoy. Follow one or more sites to get started." ) }</p>
 				</header>
 
-				<Masonry className="reader-start__cards" options={ { gutter: 14 } }>
+				{ ! hasRecommendations && this.renderLoadingPlaceholders() }
+
+				{ hasRecommendations && <Masonry className="reader-start__cards" options={ { gutter: 14 } }>
 					{ this.props.recommendationIds ? map( this.props.recommendationIds, ( recId ) => {
```
Comment (blowery) [inline client/reader/start/main.jsx]: maybe switch this to `&&` style to be consistent

**Pull request merged**